ipinfo/cmd_randip.go

```diff
@@ -26,15 +28,16 @@ func printHelpRandIP() {
 
 Description:
   Generates random IP/IPs.
-  Generates 1 IPv4 without providing any arguments. 
+  By default, generates 1 random IPv4 address, but can be configured to generate
+  any number of a combination of IPv4/IPv6 addresses.
 
   $ %[1]s randip -t ipv4 -n 5
```

Comment:
Update the example to reflect the new flag format, and please also add some more example usages.